docs/clients/cloud-watch-logs.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: client
+category: clients
+name: CloudWatchLogs
+package: async-aws/cloud-watch-logs
+---
+
+## Usage
+
+### List log streams
+
+```php
+use AsyncAws\CloudWatchLogs\CloudWatchLogsClient;
+use AsyncAws\CloudWatchLogs\Input\DescribeLogStreamsRequest;
+
+$cloudWatchLogs = new CloudWatchLogsClient();
+
+$result = $cloudWatchLogs->describeLogStreams(new DescribeLogStreamsRequest([
+    'logGroupName' => 'log-group-name',
+]));
+
+foreach ($result->getLogStreams() as $stream) {
+    echo '-'.$stream->getLogStreamName().': '.$stream->getSequenceToken().PHP_EOL;
+}
+```
+
+### Push logs
+
+```php
+use AsyncAws\CloudWatchLogs\CloudWatchLogsClient;
+use AsyncAws\CloudWatchLogs\Input\PutLogEventsRequest;
+use AsyncAws\CloudWatchLogs\ValueObject\InputLogEvent;
+
+$cloudWatchLogs = new CloudWatchLogsClient();
+
+$result = $cloudWatchLogs->putLogEvents(new PutLogEventsRequest([
+    'logGroupName' => 'log-group-name',
+    'logStreamName' => 'log-stream-name',
+    'logEvents' => [
+        new InputLogEvent([
+            'timestamp' => date('U.u') * 1000,
```

Comment:
that the kind of trick I love to see in documentation :heart:

Reply:
I cannot take credit though
https://github.com/maxbanton/cwh/blob/master/src/Handler/CloudWatch.php#L241